roles/matrix-bot-postmoogle/defaults/main.yml

```diff
@@ -0,0 +1,105 @@
+matrix_bot_postmoogle_config_path: "{{ matrix_bot_postmoogle_base_path }}/config"
+matrix_bot_postmoogle_data_path: "{{ matrix_bot_postmoogle_base_path }}/data"
+
+# A list of extra arguments to pass to the container
+matrix_bot_postmoogle_container_extra_arguments: []
+
+# List of systemd services that matrix-bot-postmoogle.service depends on
+matrix_bot_postmoogle_systemd_required_services_list: ['docker.service']
+
+# List of systemd services that matrix-bot-postmoogle.service wants
+matrix_bot_postmoogle_systemd_wanted_services_list: []
+
+
+# Database-related configuration fields.
+#
+# To use SQLite, stick to these defaults.
+#
+# To use Postgres:
+# - change the engine (`matrix_bot_postmoogle_database_engine: 'postgres'`)
+# - adjust your database credentials via the `matrix_bot_postmoogle_database_*` variables
+matrix_bot_postmoogle_database_engine: 'sqlite'
+
+matrix_bot_postmoogle_sqlite_database_path_local: "{{ matrix_bot_postmoogle_data_path }}/bot.db"
+matrix_bot_postmoogle_sqlite_database_path_in_container: "/data/bot.db"
+
+matrix_bot_postmoogle_database_username: 'postmoogle'
+matrix_bot_postmoogle_database_password: 'some-password'
+matrix_bot_postmoogle_database_hostname: 'matrix-postgres'
+matrix_bot_postmoogle_database_port: 5432
+matrix_bot_postmoogle_database_name: 'postmoogle'
+
+matrix_bot_postmoogle_database_connection_string: 'postgres://{{ matrix_bot_postmoogle_database_username }}:{{ matrix_bot_postmoogle_database_password }}@{{ matrix_bot_postmoogle_database_hostname }}:{{ matrix_bot_postmoogle_database_port }}/{{ matrix_bot_postmoogle_database_name }}?sslmode=disable'
+
+matrix_bot_postmoogle_storage_database: "{{
+	{
+		'sqlite': matrix_bot_postmoogle_sqlite_database_path_in_container,
+		'postgres': matrix_bot_postmoogle_database_connection_string,
+	}[matrix_bot_postmoogle_database_engine]
+}}"
+
+matrix_bot_postmoogle_database_dialect: "{{
+  {
+    'sqlite': 'sqlite3',
+    'postgres': 'postgres',
+  }[matrix_bot_postmoogle_database_engine]
+}}"
+
+
+# The bot's username. This user needs to be created manually beforehand.
+# Also see `matrix_bot_postmoogle_password`.
+matrix_bot_postmoogle_login: "postmoogle"
+
+# The password that the bot uses to authenticate.
+matrix_bot_postmoogle_password: ''
+
+matrix_bot_postmoogle_homeserver: "{{ matrix_homeserver_container_url }}"
+
+# Command prefix
+matrix_bot_postmoogle_prefix: '!pm'
+
+# Max email size in megabytes, including attachments
+matrix_bot_postmoogle_maxsize: '1024'
+
+# Allow room settings changes by any room participant
+matrix_bot_postmoogle_noowner: false
+
+# Allow Postmoogle use by users over federation
+matrix_bot_postmoogle_federation: false
+
+# Sentry DSN
+matrix_bot_postmoogle_sentry: ''
+
+# Log level
+matrix_bot_postmoogle_loglevel: ''
+
+# Disable encryption
+matrix_bot_postmoogle_noencryption: false
+
+matrix_bot_postmoogle_domain: "{{ matrix_server_fqn_matrix }}"
+
+matrix_bot_postmoogle_port: "25"
```

Comment:
I was wondering "why not a number here instead of a string", but I guess this is similar to the `_host_bind` varialbes we have in other roles, where it could either be a port number or an IP address + port (e.g. `127.0.0.1:25`, `0.0.0.0:25`, etc.).

---

Also, see my other comment (for `matrix-bot-postmoogle.service.j2`) about potentially redoing this in a way that doesn't use privileged ports.

Renaming it for consistency with the other _`host_bind_port` variables and adding comments like see here may also be beneficial, so that users know how to use it:

https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/5fafbab7ae4b6fd090c152144932da6e57f883cd/roles/matrix-email2matrix/defaults/main.yml#L25-L30

Of course, as of right now, `matrix_bot_postmoogle_port` is really meant to be a port number, because it's passed directly to Postmoogle as well, but that may change. See the other review comment about `matrix-bot-postmoogle.service.j2`.

Reply:
Sure. In that case I'll add additional variable, because `POSTMOOGLE_PORT` is actual env var of the postmoogle config